hazelcast/src/test/java/com/hazelcast/spi/impl/operationexecutor/classic/ClassicOperationExecutorTest.java

```diff
@@ -83,4 +85,28 @@ public void run() throws Exception {
             }
         });
     }
+
+    @Test(expected = NullPointerException.class)
+    public void test_runOnAllPartitionThreads_whenTaskNull() {
+        initExecutor();
+        executor.runOnAllPartitionThreads(null);
+    }
+
+    @Test
+    public void test_runOnAllPartitionThreads() {
+        initExecutor();
+
+        int threadCount = executor.getPartitionOperationThreadCount();
+        final CountDownLatch latch = new CountDownLatch(threadCount);
+
+        executor.runOnAllPartitionThreads(new Runnable() {
+            @Override
+            public void run() {
+                latch.countDown();
+            }
+        });
+
+        assertOpenEventually(latch);
```

Comment:
perhaps it's a nitpicking, but this test only makes sure the task is executed `threadCount`-times. Not that it's executed by all threads.